cli/command_agent_install.go

```
@@ -0,0 +1,55 @@
+			c.out.printStdout("--- %s\n%s\n", path, content)
+		}
+
+		for _, cmd := range p.Commands {
+			c.out.printStdout("$ %s\n", cmd)
+		}
+
+		return nil
+	}
+
+	return install.Apply(p, func(name string, args ...string) error {
+		cmd := exec.Command(name, args...) //nolint:gosec
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)